docs/src/tutorials/conic/tips_and_tricks.jl

```diff
@@ -338,6 +338,78 @@ set_silent(model)
 optimize!(model)
 value(t), value.(x)
 
+# ## RootDetCone
+
+# The [`MOI.RootDetConeSquare`](@ref) is a cone of the form:
+# ```math
+# K = \{ (t, X) \in \mathbb{R}^{1+d^2} : t \le \det(X)^{\frac{1}{d}} \}
+# ```
+
+model = Model(SCS.Optimizer)
+set_silent(model)
+@variable(model, t)
+@variable(model, X[1:2, 1:2])
+@objective(model, Max, t)
+@constraint(model, [t; vec(X)] in MOI.RootDetConeSquare(2))
+@constraint(model, X .== [2 1; 1 3])
+optimize!(model)
+value(t), sqrt(LinearAlgebra.det(value.(X)))
+
+# If `X` is symmetric, then you can use [`MOI.RootDetConeTriangle`](@ref)
+# instead. This can be more efficient because the solver does not need to add
```

Comment:
Can be or is more efficient?

Reply:
Can be.

It's a bit subtle: the Square->Triangle bridge will only add the equality constraints for elements that are not equivalent using `==` (or really, `isapprox`).

```julia
model = Model()
@variable(model, t)
@variable(model, x)
@variable(model, y)
@variable(model, z)

# Elements are symmetric, no constraints added 
@constraint(model, [t; vec([x y; y z])] in MOI.RootDetConeSquare(2))

# Elements are symmetric, constraints added, even though there exists a
# separate `y == w` constraint
@variable(model, w)
@constraint(model, y == w)
@constraint(model, [t; vec([x w; y z])] in MOI.RootDetConeSquare(2))
```